build/elasticsearch/Dockerfile

```diff
@@ -32,7 +32,13 @@ RUN elasticsearch-plugin install --batch x-pack
 
 COPY elasticsearch.yml /usr/share/elasticsearch/config/
 COPY log4j2.properties /usr/share/elasticsearch/config/
+COPY bin/es-docker /usr/share/elasticsearch/bin/es-docker
 
-CMD ["/bin/sh", "bin/elasticsearch"]
+USER root
```

Comment:
Yes this is layer minimization going on here :)

However, I already have that WORKDIR command that the top of the Dockerfile, so I took the opportunity to clean up the paths in https://github.com/elastic/elasticsearch-docker/commit/d875a9f8b4796e1078969d72197e915600e797d7